packages/pharos/src/components/toast/pharos-toaster.ts

```diff
@@ -1,32 +1,83 @@
 import { PharosElement } from '../base/pharos-element';
 import { html } from 'lit';
 import type { TemplateResult, CSSResultArray } from 'lit';
+import { state } from 'lit/decorators.js';
+import { repeat } from 'lit/directives/repeat.js';
 import { toasterStyles } from './pharos-toaster.css';
 
-import type { PharosToast } from './pharos-toast';
+import type { ToastStatus } from './pharos-toast';
+import { PharosToast } from './pharos-toast';
 import { DEFAULT_STATUS, DEFAULT_INDEFINITE } from './pharos-toast';
 
 import { v4 as uuidv4 } from 'uuid';
+import ScopedRegistryMixin from '../../utils/mixins/scoped-registry';
 
 /**
  * pharos-toast-open event.
  *
- * @tag pharos-toaster
```

Comment:
I think this was errantly placed here; I moved it to the component's JSDoc block below.